app/bundles/EmailBundle/Helper/EmailValidator.php

```diff
@@ -64,7 +68,19 @@ public function validate($address, $doDnsCheck = false)
      */
     public function isValidFormat($address)
     {
-        return !empty($address) && filter_var($address, FILTER_VALIDATE_EMAIL);
+        return !empty($address) && filter_var($address, FILTER_VALIDATE_EMAIL, FILTER_FLAG_EMAIL_UNICODE);
+    }
+
+    /**
+     * Validates that email has an acceptable length (254).
+     *
+     * @param string $address
+     *
+     * @return bool
+     */
+    public function isValidLength($address)
+    {
+        return strlen($address) <= 254;
```

Comment:
IMO this would need an entry in the upgrade notes too, as this changes existing behaviour and some e-mail addresses will be suddenly invalid due to length